core/server/api_container/server/startosis_engine/startosis_interpreter_test.go

```diff
@@ -62,7 +63,10 @@ type StartosisInterpreterTestSuite struct {
 func (suite *StartosisInterpreterTestSuite) SetupTest() {
 	suite.packageContentProvider = mock_package_content_provider.NewMockPackageContentProvider()
 	enclaveDb := getEnclaveDBForTest(suite.T())
-	runtimeValueStore, err := runtime_value_store.CreateRuntimeValueStore(nil, enclaveDb)
+
+	dummySerde := shared_helpers.NewDummyStarlarkValueSerDeForTest()
```

Comment:
It you push the serde down to the repository level, maybe you won't need all this since you don't need persistence for unit tests at all